src/Impostor.Hazel/MessageReader.cs

```diff
@@ -239,6 +247,19 @@ public IMessageReader Copy(int offset = 0)
             return reader;
         }
 
+        public T ReadNetObject<T>(IGame game) where T : IInnerNetObject
+        {
+            return game.FindObjectByNetId<T>(ReadPackedUInt32());
+        }
+
+        public Vector2 ReadVector2()
```

Comment:
Are we sure we want to put the lerp stuff in here instead of how it was?

Reply:
Thing is FloatRange was among us specific (this is in Impostor.Hazel) also imo it was pretty useless in the first place